src/BloomBrowserUI/bookEdit/js/readerToolsModel.ts

```diff
@@ -123,6 +123,9 @@ class ReaderToolsModel {
       ReaderToolsModel.updateElementContent("stageNumber", "0");
       return;
     }
+    if (this.stageNumber > stages.length) {
+       this.stageNumber = stages.length;
+    }
```

Comment:
Things adjust without weirdness or crashing if the selected stage is not greater than the number of stages.  I suppose in some sense if stage 3 was selected and the user deleted stages 1 and 2, the selected stage should always change to stage 1 even if we have 17 stages still in existence.  But we don't have that information available here, and I don't have a good enough understanding of the js code to achieve that happy state.